scapy/utils.py

```diff
@@ -1732,7 +1755,8 @@ def _write_packet(self,
                       sec=None,  # type: Optional[float]
                       usec=None,  # type: Optional[int]
                       caplen=None,  # type: Optional[int]
-                      wirelen=None  # type: Optional[int]
+                      wirelen=None,  # type: Optional[int]
+                      comment=None  # type: Optional[bytes]
```

Comment:
Isn't it kinda weird to have those arguments in `recv()` ?

Reply:
What do you mean?

Reply:
Sorry I misread the git diff :sweat_smile: